<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>480</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>LCoder</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="animated">
   <bool>true</bool>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>429</width>
      <height>421</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>40</height>
          </size>
         </property>
         <property name="title">
          <string>Источник</string>
         </property>
         <widget class="QWidget" name="layoutWidget">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>20</y>
            <width>141</width>
            <height>20</height>
           </rect>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QRadioButton" name="radioButton_mode_text">
             <property name="acceptDrops">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Текст</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="radioButton_mode_in_file">
             <property name="text">
              <string>Файл</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>40</height>
          </size>
         </property>
         <property name="title">
          <string>Режим</string>
         </property>
         <widget class="QWidget" name="layoutWidget">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>20</y>
            <width>202</width>
            <height>21</height>
           </rect>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QRadioButton" name="radioButton_coder">
             <property name="text">
              <string>кодирование</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="radioButton_decoder">
             <property name="acceptDrops">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>дэкодирование</string>
             </property>
             <property name="checked">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Ваше сообщщение:</string>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
         <property name="margin">
          <number>0</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="textEdit_input_text">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>425</width>
           <height>100</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>425</width>
           <height>120</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>   Ключ шифрования: </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="frame">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>30</height>
            </size>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <widget class="QWidget" name="layoutWidget">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>211</width>
              <height>31</height>
             </rect>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <item>
              <widget class="QRadioButton" name="radioButton_key_your">
               <property name="text">
                <string>свой</string>
               </property>
               <property name="checkable">
                <bool>true</bool>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
               <property name="autoExclusive">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="radioButton_key_random">
               <property name="text">
                <string>рандомно</string>
               </property>
               <property name="checkable">
                <bool>true</bool>
               </property>
               <property name="checked">
                <bool>false</bool>
               </property>
               <property name="autoExclusive">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Ключ:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_key">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="acceptDrops">
            <bool>true</bool>
           </property>
           <property name="whatsThis">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;ключ шифрования&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="frame">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="font">
          <font>
           <kerning>true</kerning>
          </font>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="text">
          <string>Шифр:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="textEdit_output_text">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>100</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>120</height>
          </size>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>500</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>о проекте</string>
    </property>
    <widget class="QMenu" name="menu_3">
     <property name="title">
      <string>Автор</string>
     </property>
     <addaction name="action_5"/>
     <addaction name="action4pda"/>
    </widget>
    <addaction name="actionGitHab"/>
    <addaction name="separator"/>
    <addaction name="menu_3"/>
   </widget>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_2">
   <property name="text">
    <string>язык</string>
   </property>
  </action>
  <action name="actionGitHab">
   <property name="text">
    <string>GitHab</string>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
  </action>
  <action name="action_5">
   <property name="text">
    <string>Вконтакте</string>
   </property>
  </action>
  <action name="action4pda">
   <property name="text">
    <string>4pda</string>
   </property>
  </action>
  <action name="action1">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>1 - для проверки программы</string>
   </property>
  </action>
  <action name="action2">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>2 - всё ещё можно взломать</string>
   </property>
  </action>
  <action name="action3">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>3 - легкий уровень защиты</string>
   </property>
  </action>
  <action name="action4">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>4 - нормальный</string>
   </property>
  </action>
  <action name="action5">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>5 - максимальный</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
